weather_app/management/commands/utils.py

```diff
@@ -0,0 +1,61 @@
+import os
+import csv
+from weather_app.models import WeatherCharacteristic,Weather,CustomTimeStampModel
+from django.core import exceptions
+
+
+""" reading a files form directory"""
+def reading_files(path):
+    all_files = []
+    """
+    loop getting files from path
+    sending files to the "all_files" list
+    """
+    for root, directory, files in os.walk(path):
```

Comment:
would be better if we use `glob` library instead of walking through os, it would be more optimized